<details>
<summary>Pre-Commit Test failed! Click here</summary>

```
[INFO] Initializing environment for git://github.com/Lucas-C/pre-commit-hooks.
[INFO] Initializing environment for git://github.com/pre-commit/pre-commit-hooks.
[INFO] Initializing environment for https://github.com/pre-commit/pre-commit-hooks.
[INFO] Initializing environment for https://github.com/adrienverge/yamllint.git.
[INFO] Installing environment for git://github.com/Lucas-C/pre-commit-hooks.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
[INFO] Installing environment for git://github.com/pre-commit/pre-commit-hooks.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
[INFO] Installing environment for https://github.com/pre-commit/pre-commit-hooks.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
[INFO] Installing environment for https://github.com/adrienverge/yamllint.git.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
Tabs remover.............................................................Failed
- hook id: remove-tabs
- exit code: 1
- files were modified by this hook

Substituting tabs in: docs/downstream/crc.md by 4 whitespaces

Tabs have been successfully removed. Now aborting the commit.
You can check the changes made. Then simply "git add --update ." and re-commit

Trim Trailing Whitespace.................................................Passed
Check for merge conflicts................................................Passed
Fix End of Files.........................................................Passed
Check for added large files..............................................Passed
Check for case conflicts.................................................Passed
Check JSON...........................................(no files to check)Skipped
Check for broken symlinks............................(no files to check)Skipped
Detect Private Key.......................................................Passed
Fix End of Files.........................................................Passed
Trim Trailing Whitespace.................................................Passed
yamllint.................................................................Failed
- hook id: yamllint
- exit code: 1

manifests/overlays/dev/serviceaccounts/argocd-dex-server.yaml
  1:1       warning  missing document start "---"  (document-start)
  10:81     error    line too long (94 > 80 characters)  (line-length)

manifests/base/routes/argocd_server_route.yaml
  1:1       warning  missing document start "---"  (document-start)

manifests/overlays/dev/applications/dev_app/kustomization.yaml
  1:1       warning  missing document start "---"  (document-start)
  4:1       error    wrong indentation: expected 2 but found 0  (indentation)

manifests/base/deployments/argocd-repo-server_patch.yaml
  1:1       warning  missing document start "---"  (document-start)
  9:7       error    wrong indentation: expected 8 but found 6  (indentation)
  12:9      error    wrong indentation: expected 10 but found 8  (indentation)
  15:9      error    wrong indentation: expected 10 but found 8  (indentation)
  22:7      error    wrong indentation: expected 8 but found 6  (indentation)

manifests/base/services/argocd_metrics_service_patch.yaml
  1:1       warning  missing document start "---"  (document-start)

manifests/overlays/dev/rbac/argocd-manager-role.yaml
  1:1       warning  missing document start "---"  (document-start)
  6:1       error    wrong indentation: expected 2 but found 0  (indentation)
  7:3       error    wrong indentation: expected 4 but found 2  (indentation)
  9:3       error    wrong indentation: expected 4 but found 2  (indentation)
  11:3      error    wrong indentation: expected 4 but found 2  (indentation)
  13:3      error    wrong indentation: expected 4 but found 2  (indentation)
  15:3      error    wrong indentation: expected 4 but found 2  (indentation)

manifests/base/services/argocd_server_metrics_service_patch.yaml
  1:1       warning  missing document start "---"  (document-start)

manifests/crds/kustomization.yaml
  1:1       warning  missing document start "---"  (document-start)
  4:1       error    wrong indentation: expected 2 but found 0  (indentation)

manifests/overlays/dev/secrets/clusters/dev.cluster.example.enc.yaml
  1:1       warning  missing document start "---"  (document-start)
  12:81     error    line too long (227 > 80 characters)  (line-length)
  21:81     error    line too long (288 > 80 characters)  (line-length)
  23:8      error    too many spaces after hyphen  (hyphens)
  23:5      error    wrong indentation: expected 8 but found 4  (indentation)

manifests/overlays/dev/secrets/clusters/secret-generator.yaml
  1:1       warning  missing document start "---"  (document-start)
  6:1       error    wrong indentation: expected 2 but found 0  (indentation)

manifests/overlays/dev/projects/projects.yaml
  1:1       warning  missing document start "---"  (document-start)
  7:3       error    wrong indentation: expected 4 but found 2  (indentation)
  10:3      error    wrong indentation: expected 4 but found 2  (indentation)

manifests/base/serviceaccounts/argocd-manager.yaml
  1:1       warning  missing document start "---"  (document-start)

manifests/overlays/dev/projects/kustomization.yaml
  1:1       warning  missing document start "---"  (document-start)
  4:1       error    wrong indentation: expected 2 but found 0  (indentation)

examples/argocd-manager-rolebinding.yaml
  1:1       warning  missing document start "---"  (document-start)

examples/argocd-cluster-binding.yaml
  1:1       warning  missing document start "---"  (document-start)

manifests/overlays/dev/rbac/argocd-manager-role-binding.yaml
  1:1       warning  missing document start "---"  (document-start)
  10:1      error    wrong indentation: expected 2 but found 0  (indentation)

manifests/base/deployments/argocd-server_patch.yaml
  1:1       warning  missing document start "---"  (document-start)
  9:7       error    wrong indentation: expected 8 but found 6  (indentation)
  11:9      error    wrong indentation: expected 10 but found 8  (indentation)

manifests/base/kustomization.yaml
  1:1       warning  missing document start "---"  (document-start)
  4:1       error    wrong indentation: expected 2 but found 0  (indentation)
  8:1       error    wrong indentation: expected 2 but found 0  (indentation)

manifests/overlays/dev/kustomization.yaml
  1:1       warning  missing document start "---"  (document-start)
  5:1       error    wrong indentation: expected 2 but found 0  (indentation)
  11:1      error    wrong indentation: expected 2 but found 0  (indentation)
  25:1      error    wrong indentation: expected 2 but found 0  (indentation)
  29:1      error    wrong indentation: expected 2 but found 0  (indentation)

manifests/overlays/dev/applications/dev_app/dev-app.yaml
  1:1       warning  missing document start "---"  (document-start)
  16:5      error    wrong indentation: expected 6 but found 4  (indentation)

manifests/overlays/dev/secrets/gpg/secret-generator.yaml
  1:1       warning  missing document start "---"  (document-start)
  6:1       error    wrong indentation: expected 2 but found 0  (indentation)

.aicoe-ci.yaml
  1:1       warning  missing document start "---"  (document-start)

manifests/overlays/dev/applications/kustomization.yaml
  1:1       warning  missing document start "---"  (document-start)
  4:1       error    wrong indentation: expected 2 but found 0  (indentation)

.sops.yaml
  1:1       warning  missing document start "---"  (document-start)

examples/odh-deployment-app.yaml
  1:1       warning  missing document start "---"  (document-start)
  19:5      error    wrong indentation: expected 6 but found 4  (indentation)

manifests/overlays/dev/secrets/clusters/.sops.yaml
  1:1       warning  missing document start "---"  (document-start)

examples/odh-operator-app.yaml
  1:1       warning  missing document start "---"  (document-start)
  19:5      error    wrong indentation: expected 6 but found 4  (indentation)

manifests/crds/aggregate-to-admin.yaml
  1:1       warning  missing document start "---"  (document-start)
  8:1       error    wrong indentation: expected 2 but found 0  (indentation)
  9:3       error    wrong indentation: expected 4 but found 2  (indentation)
  11:3      error    wrong indentation: expected 4 but found 2  (indentation)
  14:3      error    wrong indentation: expected 4 but found 2  (indentation)
  30:1      error    wrong indentation: expected 2 but found 0  (indentation)
  31:3      error    wrong indentation: expected 4 but found 2  (indentation)
  33:3      error    wrong indentation: expected 4 but found 2  (indentation)
  35:3      error    wrong indentation: expected 4 but found 2  (indentation)

manifests/overlays/dev/secrets/auth/secret-generator.yaml
  1:1       warning  missing document start "---"  (document-start)
  6:1       error    wrong indentation: expected 2 but found 0  (indentation)

manifests/overlays/dev/secrets/auth/dex-server-sa.enc.yaml
  1:1       warning  missing document start "---"  (document-start)
  15:81     error    line too long (288 > 80 characters)  (line-length)
  17:8      error    too many spaces after hyphen  (hyphens)
  17:5      error    wrong indentation: expected 8 but found 4  (indentation)

manifests/overlays/dev/secrets/auth/argocd-secret.enc.yaml
  1:1       warning  missing document start "---"  (document-start)
  9:81      error    line too long (207 > 80 characters)  (line-length)
  10:81     error    line too long (160 > 80 characters)  (line-length)
  11:81     error    line too long (1432 > 80 characters)  (line-length)
  18:81     error    line too long (288 > 80 characters)  (line-length)
  20:8      error    too many spaces after hyphen  (hyphens)
  20:5      error    wrong indentation: expected 8 but found 4  (indentation)

manifests/base/deployments/argocd-redis_patch.yaml
  1:1       warning  missing document start "---"  (document-start)
  19:7      error    wrong indentation: expected 8 but found 6  (indentation)
  20:9      error    wrong indentation: expected 10 but found 8  (indentation)
  28:9      error    wrong indentation: expected 10 but found 8  (indentation)

manifests/overlays/dev/secrets/gpg/secret.enc.yaml
  1:1       warning  missing document start "---"  (document-start)
  6:81      error    line too long (6448 > 80 characters)  (line-length)
  13:81     error    line too long (288 > 80 characters)  (line-length)
  15:8      error    too many spaces after hyphen  (hyphens)
  15:5      error    wrong indentation: expected 8 but found 4  (indentation)
```
</details>

--- a/docs/downstream/crc.md
+++ b/docs/downstream/crc.md
@@ -22,7 +22,7 @@ As an alternative you can install the prerequisites locally:
  * Fork https://github.com/operate-first/continuous-deployment
 
  * Import GPG key that is used by kustomize to encrypt the secrets.\
-    ```base64 -d < examples/key.asc | gpg --import ```
+   ```base64 -d < examples/key.asc | gpg --import ```
 
  * Proceed with [setup_argocd_dev_environment.md](../setup_argocd_dev_environment.md)
 

--- a/docs/downstream/odh-install-crc.md
+++ b/docs/downstream/odh-install-crc.md
@@ -25,7 +25,7 @@ Our ArgoCD SA `argocd-manager` needs to have access to the new project. On CRC I
 Then you need to change the `dev-cluster` to include all projects/namespace. (This is done by actually removing the value of `namespace`).
 
 ```bash
-oc patch secret dev-cluster-spec -n aicoe-argocd-dev --type='json' -p="[{'op': 'add', 'path': '/data/namespace', 'value':''}]"
+oc patch secret dev-cluster-spec -n aicoe-argocd-dev --type='json' -p="[{'op': 'replace', 'path': '/data/namespaces', 'value':''}]"
 ```
 
 Be aware that if you change this value in the ArgoCD UI, you might loose the stored credentials for the cluster due to a bug in ArgoCD.

--- a/examples/odh-deployment-app.yaml
+++ b/examples/odh-deployment-app.yaml
@@ -1,5 +1,7 @@
 apiVersion: argoproj.io/v1alpha1
 kind: Application
+metadata:
+  name: odh-deployment
 spec:
   destination:
     namespace: odh-deployment

--- a/examples/odh-operator-app.yaml
+++ b/examples/odh-operator-app.yaml
@@ -1,5 +1,7 @@
 apiVersion: argoproj.io/v1alpha1
 kind: Application
+metadata:
+  name: odh-operator
 spec:
   destination:
     namespace: odh-operator

